internal_ws/yktrace/src/hwt/mapper.rs

```diff
@@ -1,3 +1,6 @@
+//! The mapper loads SIR location labels from the executable and provides a way to map virtual
+//! block addresses to SIR locations, i.e. symbol name and block index.
```

Comment:
"i.e." or "e.g."?

Reply:
"i.e." as in a SIR location is made up of a symbol name and block index.